pkg/granted/registry/template.go

```diff
@@ -0,0 +1,30 @@
+package registry
+
+import (
+	"bytes"
+	"text/template"
+)
+
+type Template struct {
+	RepoURL string
+}
+
+const AUTO_GENERATED_MSG string = `### GRANTED-REGISTRY-SECTION: "{{.RepoURL}}". DO NOT EDIT.
```

Comment:
nitpick: can you make this match the autogenerated description in the RFD? This is very very minor but I think that the uppercase is less aesthetically nice than `# granted-start: https://github.com/octo-org/granted-registry.git`

https://github.com/common-fate/rfds/discussions/2

```suggestion
const AUTO_GENERATED_MSG string = `### GRANTED-REGISTRY-SECTION: "{{.RepoURL}}". DO NOT EDIT.
```

Reply:
@chrnorm We might need to remove `repoURL` from the autogenerated description cause we can have multiple repos now.